config/ftbquests/quests/chapters/immersive_engineering.snbt

```diff
@@ -1380,5 +1386,66 @@
 				player_command: false
 			}]
 		}
+		{
+			x: 0.5d
+			y: -4.5d
+			description: [
+				"Item batcher is advanced filter block made specifically for use with conveyor belts and creating batches of items for machines."
+				""
+				"It has a lot of filtering options, few modes of output (Single - Works like normal filter, if item matches it, will be redirected to output / Batch mode - All filters needs to be satisfied for all items to be redirected to the output) and emits Redstone depending on the state of Filters."
+				""
+				"Check Engineer's Manual for more information."
+			]
+			dependencies: ["000000000000001E"]
+			min_width: 300
+			id: "215AD1198A1BF8D0"
+			tasks: [{
+				id: "003B96D16776DCBA"
+				type: "item"
+				item: "immersiveengineering:item_batcher"
+			}]
+			rewards: [{
+				id: "70A61D684AF8B5FF"
+				type: "command"
+				title: "Rare Immersive Engineering Loot Box"
+				icon: "kubejs:rare_lootbox"
+				command: "/execute at @p run loot spawn ~ ~1 ~ loot enigmatica:loot_boxes/quest_immersive_engineering_loot_rare"
+				player_command: false
+			}]
+		}
+		{
+			x: 1.5d
+			y: -6.0d
+			description: [
+				"Item Router is advanced item sorter made to work with conveyor belts."
+				""
+				"It can output items on any side, as well get items from every side. However item can't be outputted to the side it was inserted from. If item doesn't match any filter, it will be outputted on not filtered side."

```

Comment:
It can in- and output items on every side. However items can't be outputted to the same side as it was inserted from. If an item doesn't match any filter, it will be outputted on a non-filtered side.

Personally I prefer `output` over `outputted`, but both are grammatically correct according to a brief google search :P